Improve API of animation (#3878)
cyberbotics/webots: docs/guide/web-animation.md

The web animation is played by a web component from the [WebotsAnimation.js] package called `webots-animation`.

The following attributes are available:

> * `x3d`: the name of the .x3d file containing the animation world.

Reviewer: ```suggestion
* `x3d`: the name of the .x3d file containing the 3d model.
```

> * `json`: the name of the .json file containing the animation steps.

Reviewer: ```suggestion
* `json`: the name of the .json file containing the animation sequence.
```
That seems more appropriate, see for example https://docs.unrealengine.com/4.27/en-US/AnimatingObjects/SkeletalMeshAnimation/Sequences/

> * `autoplay`: 'true' or 'false' to determine if the animation should be played automatically. If `x3d` is set, the animation will be played by default.

Reviewer: It should be related with the animation file, not the 3D file, isn't it?
```suggestion
* `autoplay`: 'true' or 'false' to determine if the animation should be played automatically.
If `json` is set, the animation will be played by default.
```

For more complex interaction with the web component, the following functions are available:
* `active()`: return `true` if there is already a animation loaded by the web component, `false` otherwise.
* `close()`: close the current animation.

> * `setX3d(fileName)`: set the name of the x3d file for the next animation to be loaded.

Reviewer: ```suggestion
* `setX3d(fileName)`: set the name of the x3d file for the next model to be loaded.
```

>   * `fileName`: the name of the x3d file.

Reviewer: ```suggestion
  * `fileName`: name of the x3d file.
```

* `setJson(fileName)`: set the name of the json file for the next animation to be loaded.

>   * `fileName`: the name of the json file.

Reviewer: ```suggestion
  * `fileName`: name of the json file.
```
